pkg/controllers/collaset/synccontrol/sync_control.go

```diff
@@ -431,7 +480,7 @@ func (r *RealSyncControl) Update(
 	// 3. prepare Pods to begin PodOpsLifecycle
 	podCh := make(chan *PodUpdateInfo, len(podToUpdate))
 	for i, podInfo := range podToUpdate {
-		if podInfo.IsUpdatedRevision && !podInfo.PodDecorationChanged {
+		if podInfo.IsUpdatedRevision && !podInfo.PodDecorationChanged && !podInfo.isInReplacing {
```

Comment:
The logic here is not clear enough.
Can we move the `!podInfo.isInReplacing` out of this IF statement?